engine/runtime/src/main/java/org/enso/interpreter/instrument/ReplExecutionEventNode.java

```diff
@@ -0,0 +1,44 @@
+package org.enso.interpreter.instrument;
+
+import scala.util.Either;
+
+import java.util.Map;
+
+public interface ReplExecutionEventNode {
```

Comment:
I'd be interested in knowing where I can get access to this node from? What API it is associated with?

Reply:
Not sure I understand what you mean, but `DebuggerMessageHandler` was using the nested class, hence the extraction of the interface that can safely stay in `runtime` project.